packages/events/defaultSubscribers/compile.js

```diff
@@ -62,11 +62,11 @@ module.exports = {
         this.logger.log(`${OS.EOL}    ${infos.join()}`);
       }
     ],
-    "compile:nothingToCompile": [
```

Comment:
We can't just get rid of this! With compilation, we might have "nothing to compile", or we might be skipping. There are two separate situations here.